.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: go
+
+go:
+  - 1.9.2
+
+env:
+  global:
+   - ARCH=$(go env GOARCH)
+   - TAG=$(git describe --tags --always)
+
+sudo: required
+
+services:
+  - docker
+
+install:
+  - make bootstrap
+
+script:
+  - make && make test
+
+after_success:
+  - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
+      export REPO=openebs/node-disk-manager-$ARCH:$TAG;
+      docker login -u "$DNAME" -p "$DPASS";
+      docker push $REPO;
+    fi
+  - if [ -n "$TRAVIS_TAG" ]; then
+      export REPO=openebs/node-disk-manager-$ARCH:$TAG;
```

Comment:
yes kmova, it will check if a tag is pushed for example v0.1 since travis stores the tag in this variable so then also it will push the image to dockherhub.